resources/views/admin/dashboard_all_reservations.blade.php

```diff
@@ -0,0 +1,63 @@
+@extends('layouts.app')
+
+@section('content')
+
+{{-- temporaly use for front end coding--}}
+               <!-- Bootstrap -->
+               <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
```

Comment:
We can remove the import of bootstrap here because laravel has bootstrap already installed.

Reply:
for the the link tag @kayo-24 you can implement same with my comment here
https://github.com/kredo-school/14th_explore_ease/pull/14#discussion_r1383161110

Reply:
I fixed.